rasa/cli/arguments/data.py

```diff
@@ -67,3 +68,56 @@ def set_validator_arguments(parser: argparse.ArgumentParser):
     )
     add_domain_param(parser)
     add_data_param(parser)
+
+
+def set_augment_arguments(parser: argparse.ArgumentParser):
+    add_config_param(parser)
+
+    parser.add_argument(
+        "--nlu-training-data", type=str, help="File containing your NLU training data."
+    )
+
+    parser.add_argument(
+        "--nlu-evaluation-data",
+        type=str,
+        help="File containing your NLU evaluation data.",
+    )
+
+    parser.add_argument(
+        "--nlu-classification-report",
+        type=str,
+        help="File containing your NLU classification report.",
+    )
+
+    parser.add_argument(
+        "--paraphrases", type=str, help="File containing your paraphrases."
+    )
+
+    parser.add_argument(
+        "--num-intents",
+        type=int,
+        default=5,
+        help="Number of intents per performance criterion for which to perform data augmentation.",
+    )
+
+    parser.add_argument(
+        "--paraphrase-score-threshold",
+        type=float,
+        default=0.8,
```

Comment:
Have we played with this default?

Reply:
No 😅 . 

Wdyt is the best way forward? Run some tests with e.g. financial-demo or whatever we have floating around, or delete the default value?

Reply:
We shouldn't delete it because users won't have any idea on how to tune it. It's okay to leave it to 5 for now but let's have a follow up issue on investigating a good value for the parameter. Can you create a follow up issue please?